clang/tools/driver/cc1_main.cpp

```diff
@@ -245,10 +245,10 @@ int cc1_main(ArrayRef<const char *> Argv, const char *Argv0, void *MainAddr) {
     llvm::timeTraceProfilerWrite(*profilerOutput);
     llvm::timeTraceProfilerCleanup();
 
-    llvm::errs() << "Time trace json-file dumped to " << Path.str() << "\n";
-    llvm::errs()
-        << "Use chrome://tracing or Speedscope App "
-           "(https://www.speedscope.app) for flamegraph visualization\n";
+//    llvm::errs() << "Time trace json-file dumped to " << Path.str() << "\n";
+//    llvm::errs()
+//        << "Use chrome://tracing or Speedscope App "
+//           "(https://www.speedscope.app) for flamegraph visualization\n";
```

Comment:
Commenting out (rather than removing altogether or wrapping in `#if SOMETHING`) makes this look like a very temporary change. What's the thinking?

Reply:
We've discussed proposing to remove this upstream as it's not very in keeping with the normal "no news is good news" style of compiler output. This was just intended as a quick fix for this branch.

Reply:
That makes sense.